webmagic-selenium/src/test/java/us/codecraft/webmagic/script/selenium/ScriptSeleniumWindowsChromeTest.java

```diff
@@ -0,0 +1,73 @@
+package us.codecraft.webmagic.script.selenium;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.Arrays;
+
+import org.junit.Test;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+
+/**
+ * 
+ * @author yang
+ *
+ */
+public class ScriptSeleniumWindowsChromeTest {
+
+    //@Ignore("need chrome driver")
+    @Test
+    public void testExcuteScript() {
+      
+    	System.getProperties().setProperty("webdriver.chrome.driver", "D:\\software\\chrome\\chromedriver.exe");
```

Comment:
Does this work in CI cycle?

Reply:
Pass the junit test。But should push the chromedriver.exe file and modify the code as System.getProperties().setProperty("webdriver.chrome.driver", "classpath\\chromedriver.exe").
I will do this later.

Reply:
and then non-Windows?

Reply:
Run on windows system as backstage process.